architecture/core_objects/routing.adoc

```diff
@@ -316,23 +345,29 @@ available in OpenShift.
 The HAProxy template router implementation is the reference implementation for a
 template router plug-in. This uses the `openshift/origin-haproxy-router`
 repository to run an HAProxy instance alongside the template router plug-in. To
-test routes, an install script is provided in the *_hack/install-router.sh_*
-file.
+test routes, an install command is provided.
 
-The router script requires two parameters: the router ID and the full URL to the
-master. It attempts to create the router based on the generated JSON file if it
-can find the `osc` executable on the path. If it cannot find the executable, it
-creates the JSON file and displays the location. You can then manually run the
-create command.
+----
+Examples:
+  Check the default router ("router"):
 
-.Running the Router Script
-====
+  $ openshift ex router
```

Comment:
Yes, they're functionally equivalent,  I just always used openshift ex.  Updating and updating my brain cache to save 7 chars every day.  I type this a lot!